code/modules/status_system/statusEffects.dm

```diff
@@ -2451,7 +2451,43 @@
 /datum/statusEffect/loose_brain
 	id = "loose_brain"
 	name = "Loose Brain"
-	desc = "You get the feeling that fliping with your brain exposed might not be a good idea..."
+	desc = "You get the feeling that flipping with your brain exposed might not be a good idea..."
 	icon_state = "brain"
 	maxDuration = 2 MINUTES // I made this long so you can do gags where you fling your brain at someone
 	effect_quality = STATUS_QUALITY_NEGATIVE
+
+/datum/statusEffect/smellingsalts //Status effect from inhaling smelling salts
+	id = "smelling_salts"
+	name = "Perked up"
+	desc = "Smelling salts have knocked you back into being awake!"
+	icon_state = "smelling_salts"
+	maxDuration = 6 MINUTES
+	effect_quality = STATUS_QUALITY_POSITIVE
+	var/max_health_bonus = 30
+	var/benefit_duration = 60 // how long the positives apply
+	var/current_bonus = 0
+
+	getTooltip()
+		if (duration > 5 MINUTES)
+			return "Smelling salts have knocked you back into being awake!"
+		else
+			return "Your sinuses are burning! Smelling salts can't perk you up."
+
+	onUpdate(optional=null)
+		var/seconds_left = round(duration/10)
+		var/cooldown = (maxDuration/10 - benefit_duration)
+		var/bonusRemaining = max(0,(seconds_left - cooldown)/benefit_duration)
+		affectHealth(round(max_health_bonus  * bonusRemaining))
+		if(bonusRemaining == 0)
+			icon_state = "smelling_salts_low"
+		else
+			icon_state = "smelling_salts"
+		return
+
+	proc/affectHealth(var/newBonus)
+		if (current_bonus != newBonus)
+			var/change = newBonus - current_bonus
+			var/mob/M = owner
+			M.max_health += change
+			current_bonus = newBonus
+			health_update_queue |= M
```

Comment:
Should either typecheck somewhere so this won't runtime if put on a non-mob, or filter to mobs in `preCheck()`